src/binary/ibc-setup/utils/resolve-required-option.spec.ts

```diff
@@ -0,0 +1,20 @@
+import test from 'ava';
+
+import { resolveRequiredOption } from './resolve-required-option';
+
+test('uses resolveOption func', (t) => {
+  const option = resolveRequiredOption(
+    'some option', // identifier
+    undefined,
+    undefined,
+    'some string',
+    'another string'
+  );
+  t.is(option, 'some string');
+});
```

Comment:
Since `resolveRequiredOption` is a proxy to `resolveOption`, I put this test here to make sure that everything's okay.